Islandora/islandora_scholar
# Assimilate Islandora Google Scholar in to Islandora Scholar

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2448

# What does this Pull Request do?
Moves all Islandora Google Scholar functionality into Islandora Scholar module, along with an update hook to disable the Islandora Google Scholar module.

# What's new?
Islandora Google Scholar's variables were already included in Islandora Scholar's .install file uninstall hook, so no work needed there.

Islandora Google Scholar's utility functions for building and embedding meta tags was moved into islandora_scholar/includes/google_scholar.inc.

Those functions got called in Islandora Google Scholar's CMODEL_PID_view_object() hooks, these are now in the same place but in islandora_scholar.module's CMODEL_PID_view_object() hooks.

# How should this be tested?
Boot up a VM with the current 7.x HEAD Scholar module, then pull this PR. You should immediately see error messages about the Islandora Google Scholar module being missing. Run /update.php, and this error message should go away. View an object with robust metadata (insufficient metadata will stop the meta tags from displaying, its always been this way) and verify that the meta tags are in the HTML source.

# Additional Notes:
This PR supercedes https://github.com/Islandora/islandora_scholar/pull/325

# Interested parties
@Islandora/7-x-1-x-committers @bondjimbond @DonRichards @jonathangreen
